yew/src/html/scope.rs

```diff
@@ -172,14 +176,23 @@ impl<COMP: Component> Scope<COMP> {
 
     /// Send a batch of messages to the component.
     ///
-    /// This is useful for reducing re-renders of the components because the messages are handled
-    /// together and the view function is called only once if needed.
+    /// This is useful for reducing re-renders of the components
+    /// because the messages are handled together and the view
+    /// function is called only once if needed.
+    ///
+    /// Please be aware that currently this method
+    /// immediately/synchronously schedules calls to the
```

Comment:
nit: I think simply stating "synchronously" is sufficient. Communicating that it sometimes sends immediately doesn't add much more to the understanding. Thoughts?